packages/notifi-dapp-example/src/app/page.tsx

```diff
@@ -29,6 +31,23 @@ export default function Home() {
 
   return (
     <main className="flex min-h-screen flex-col items-center justify-center inter.className">
+      <div>{selectedWallet && JSON.stringify(wallets[selectedWallet])}</div>
+      <div
```

Comment:
is this section for test purpose?

Reply:
Yup, this are very first test version. 
Just realize we better not to merge it to our common stage branch `stage-open-defi`. 
Created a new stage based on `stage-open-defi` called `stage-open-defi-metamask` to which our new wallet provider PRs will merge
@marukohao